sli/CMakeLists.txt

```diff
@@ -122,6 +129,13 @@ install( TARGETS sli_readline sli_lib sli
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     )
+else ( static-libraries )
+install( TARGETS sli_lib
```

Comment:
This looks a bit surprising at first, but looking at lines 116-119 above, I assume this installs the `sli` executable? It would maybe be a good idea to add a comment about this.

Reply:
I am confused by your comment. If there is no sli executable when linking statically, why do we need the else-part? I thought the purpose of the else-part was to install the sli executable. I may be mistaken here, it is a while since I built NEST statically.